spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -256,23 +255,12 @@ public SparkContext createSparkContext() {
 
     String classServerUri = null;
 
-    try { // in case of spark 1.1x, spark 1.2x
-      Method classServer = interpreter.intp().getClass().getMethod("classServer");
```

Comment:
What is the reason R interpreter (subject of this PR) requires this change?
What is going to happen on Spark 1.1x and spark 1.2x where this method is called 'classServer'  [SparkIMain.scala#L106](https://github.com/apache/spark/blob/branch-1.1/repl/src/main/scala/org/apache/spark/repl/SparkIMain.scala#L106)